Binary files a/cjs-devel-128.1-1.fc43.x86_64.rpm and b/cjs-devel-128.1-1.fc43.x86_64.rpm differ
diff -uNrp a/meson.build b/meson.build
--- a/meson.build	2025-12-10 15:40:57.000000000 +0000
+++ b/meson.build	2026-01-21 10:42:13.388115407 +0000
@@ -600,6 +600,7 @@ pkg.generate(libcjs, name: api_name, des
         'exec_prefix=${prefix}',
         'datarootdir=${datadir}',
         'cjs_console=${bindir}/cjs-console',
+        'mozjs_dep_name=@0@'.format(spidermonkey.name()),
     ])
 
 ### Test environment ###########################################################
